src/Common/src/Interop/Unix/System.Net.Http.Native/Interop.VersionInfo.cs

```diff
@@ -47,8 +48,67 @@ internal enum CurlFeatures : int
         [DllImport(Libraries.HttpNative, EntryPoint = "HttpNative_GetSslVersionDescription")]
         internal static extern string GetSslVersionDescription();
 
-        internal const string OpenSsl10Description = "openssl/1.0";
+        internal const string OpenSslDescriptionPrefix = "OpenSSL/";
         internal const string SecureTransportDescription = "SecureTransport";
         internal const string LibreSslDescription = "LibreSSL";
+
+#if !SYSNETHTTP_NO_OPENSSL
```

Comment:
It looks like this is defined in the tests project?  The tests pull in this file?  Ugh, but ok.